src/validations.ts

```diff
@@ -202,4 +202,28 @@ Then(
     }
 );
 
+/**
+ * Verify that css property of element satisfies condition
+ * @param {string} property - property to verify
+ * @param {string} alias - element to verify
+ * @param {string} validationType - validation
+ * @param {string} value - expected value
+ * @example I expect 'color' css property of 'Search Input' to be equal 'rgb(42, 42, 42)'
+ * @example I expect 'font-family' css property of 'Label' to contain 'Fira'
+ */
+Then(
+    'I expect {string} css property of {string} {wdioValidation} {string}',
+    async function (property: string, alias: string, validationType: string, value: string) {
+        const propertyName = await getValue(property);
+        const expectedValue = await getValue(value);
+        const element = await getElement(alias) as ElementAsync;
+        const validation = getValidation(validationType);
+        await conditionWait(element, conditionValidations.PRESENT, config.browser.timeout.present);
+        const actualValue = await browser.execute(function (element: any, propertyName: any) {
```

Comment:
same question here, I guess we know types of `element` and `propertyName` variables

Reply:
I updated types and replaced key access with getPropertyValue method